.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
why placing this file inside `frontend` dir when it is already in `yangacatalog-ui`

Reply:
I've had debug issues while using file inside yangcatalog-ui